ArduPlane/quadplane.cpp

```diff
@@ -461,6 +460,20 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("TKOFF_ARSP_LIM", 15, QuadPlane, maximum_takeoff_airspeed, 0),
 
+    // @Param: FWD_THR_CH
+    // @DisplayName: VTOL forward throttle channel
+    // @Description: RC channel for manual control of VTOL forward throttle.
+    // @Range 0 16
+    // @RebootRequired: True
+    AP_GROUPINFO("FWD_THR_CH", 16, QuadPlane, fwd_thr_chan, 0),
```

Comment:
I think its better to try and move to a RC_Option for this, you define the option and just use it as a label rather than do anything with it in RC_Channels, see how the Mainsail input (207) works on rover

Reply:
Thanks for the suggestion; eliminates that parameter and saves a line of code, I think.